dkim.js

```diff
@@ -241,17 +241,17 @@ class DKIMObject {
         if (this.run_cb) return;
 
         const bh = this.bh.digest('base64');
-        this.debug(this.identity + ':' +
-            ' bodyhash=' + this.fields.bh +
-            ' computed=' + bh);
+        this.debug(`${this.identity}:
+             bodyhash=${this.fields.bh}
+             computed=${bh}`);
```

Comment:
in template literals, all that leading whitespace is now part of the string. Oops.